grabber/main.go

```diff
@@ -144,6 +145,14 @@ func main() {
 	}
 	logger.Info("Stopping")
 }
+func migrator(ctx context.Context, importer *backend.Backend, lgr *zap.Logger) {
+	err := importer.MigrateDB(ctx, lgr)
+	if err != nil {
+		importer.Lgr.Fatal("Migration failed", zap.Error(err))
```

Comment:
Is this truly fatal? Wouldn't we prefer to have the old format continue to be imported rather than go in to a restart-crash loop where no new data is imported?

Reply:
And should both log lines be using the same logger?